3rdparty/python/requirements.txt

```diff
@@ -36,5 +36,6 @@ pyjwt
 pytest
 requests
 redis
+setuptools
```

Comment:
This is required since pulumi_aws apparently needs pkg_resources

Reply:
I wonder if this is somehow related to https://github.com/grapl-security/grapl/blob/dd8ba3bfd442f95b88d64d4fdac5b3e8a4959d7c/build-support/manage_virtualenv.sh#L54-L61

🤔

Reply:
So after thinking about this for a while, I don't think so. If this was the problem then I'd expect to see setuptools in the pants cache directory even if pkg_resources was not in there. My guess is with a properly updated constraints file I might be able to remove this